CHANGELOG.md

```diff
@@ -7,9 +7,9 @@
 
 ## Latest
 * Update Razor to 7.0.0-preview.23611.2 (PR: [#6741](https://github.com/dotnet/vscode-csharp/pull/6741)
-
-## 2.14.?
 * Update Roslyn to 4.9.0-3.23611.3 (PR: [#6736](https://github.com/dotnet/vscode-csharp/pull/6736))
```

Comment:
I made a silly mistake and didn't re-pull from main after this PR merged.  So it's not actually in the release branch.  Not a big deal in this case - there  wasn't any extremely important VSCode bugs in the update.

Reply:
Assuming the v2.14.8 tag is correct, yes, this matches current history.